react/features/base/conference/middleware.js

```diff
@@ -13,7 +18,12 @@ import {
     _setAudioOnlyVideoMuted,
     setLastN
 } from './actions';
-import { SET_AUDIO_ONLY, SET_LASTN } from './actionTypes';
+import {
+    LOCK_STATE_CHANGED,
+    SET_AUDIO_ONLY,
+    SET_LASTN,
+    SET_PASSWORD_FAILED
```

Comment:
Shouldn't LOCK_STATE_CHANGED AND SET_PASSWORD_FAILED be part of the lock-room middleware? @lyubomir should we try to keep those within the feature?